ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.11.yaml

```diff
@@ -57,6 +57,12 @@ tests:
   steps:
     cluster_profile: vsphere
     workflow: cucushift-installer-rehearse-vsphere-ipi-baselinecaps-none-additionalenabledcaps
+- as: installer-rehearse-nutanix
+  cluster: build01
+  interval: 960h
+  steps:
+    cluster_profile: nutanix-qe
```

Comment:
In `nutanix-qe` vault collection, I did not see `pull-secret`, `ssh-privatekey` and `ssh-publickey`, I did not see `cluster-secrets-nutanix-qe` cluster profile definition either, how the installation got those value when creating install-config.yaml on ipi-conf ref.

Reply:
some values come from https://github.com/openshift/release/pull/31648/files, it's still use dev's ssh-privatekey and ssh-publickey. I'm wondering if it's OK to replace QE ssh key directly in vault? Or I need to delete the ssh key in core-services/ci-secret-bootstrap/_config.yaml first.

Reply:
seems like it will conflicts if place ssh key both in core-services/ci-secret-bootstrap/_config.yaml and vault. I'll remove it from _config.yaml, and place QE ssh key under vault.

Reply:
Oh, sorry, I did not grep the existing well. For ssh key, yeah, we should remove `ssh-privatekey` and `ssh-publickey` setting from `cluster-secrets-nutanix-qe` in core-services/ci-secret-bootstrap/_config.yaml, and place them in openshift-qe vault.

Reply:
> seems like it will conflicts if place ssh key both in core-services/ci-secret-bootstrap/_config.yaml and vault

Right, we need to firstly update  core-services/ci-secret-bootstrap/_config.yaml get merged, then place the keys in vault.

Reply:
Done, PR https://github.com/openshift/release/pull/31928 merged